common/persistence/persistence-tests/persistenceTestBase.go

```diff
@@ -111,13 +111,13 @@ const (
 )
 
 // NewTestBaseFromParams returns a customized test base from given input
-func NewTestBaseFromParams(params TestBaseParams) TestBase {
+func NewTestBaseFromParams(params TestBaseParams) *TestBase {
 	logger, err := loggerimpl.NewDevelopment()
 	if err != nil {
 		panic(err)
 	}
 
-	return TestBase{
+	return &TestBase{
```

Comment:
looks like some other tests depend on this too. unit tests are failing.

Reply:
Yes, it turns out that most of persistence tests use this and copy suite all the time. Fixed